### ISSUE TITLE
Eingabefeld Studiendauer in Semester

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/FAU-Studium#114]
Aktuell wird hier das falsche Eingabefeld verwendet, nämlich eine Textarea:

<img width="1835" alt="grafik" src="https://github.com/RRZE-Webteam/FAU-Studium/assets/1070669/0b147bde-1bf6-4ce1-b64d-9e96f8dc798f">

Wenn nur eine Zahl eingegeben werden soll, wäre es besser auch nur ein numerisches Eingabefeld zu haben.

**Comment by wrcg** [comment]
nein, bitte so lassen - bei den Lehramtsstudiengängen gibt es je nach Lehramtsart eine andere Zahl, deswegen haben wir das Feld als Textfeld.

**Comment by xwolfde** [comment]
Aber es ist doch auch dann eine Zahl und kein Textabsatz?

**Comment by wrcg** [comment]
![image](https://github.com/RRZE-Webteam/FAU-Studium/assets/9526974/28473e83-be85-48ef-b980-176ac29b950a)

so siehts aktuell aus - das geht nicht mit einem Feld für Zahlen, oder?

**Comment by xwolfde** [comment]
Dann ist es aber auch nur ein einzeiliges Texteingabefeld, keine Textarea

**Comment by wrcg** [comment]
dann gerne so!

**Comment by xwolfde** [comment]
ok, ich schreibs oben rein, damit wir einen klaren issue haben

**Comment by xwolfde** [comment]
https://inpsyde.atlassian.net/jira/software/c/projects/FAU/boards/538?selectedIssue=FAU-385
